pkg/controller/networkpolicy/validate_test.go

```diff
@@ -1509,12 +1595,14 @@ func TestValidateAntreaClusterGroup(t *testing.T) {
 func TestValidateAntreaGroup(t *testing.T) {
 	tests := []struct {
 		name           string
-		group          *crdv1alpha3.Group
+		currGroup      *crdv1alpha3.Group
```

Comment:
Usually we define it with perfix `cur` instead of `curr`.
```suggestion
		curGroup      *crdv1alpha3.Group
```